docs/user/config/index.md

```diff
@@ -27,27 +39,35 @@ module.exports = {
 {
   // [...]
   "jest": {
+    // Replace `ts-jest` with the preset you want to use
+    // from the above list
     "preset": "ts-jest"
   }
 }
 ```
 
 </div></div>
 
+**Note:** presets are using `testMatch`, as Jest's does in its defaults. If you want to use `testRegex` instead in your configuration, you MUST set `testMatch` to `null` or Jest will bail.
```

Comment:
`presets are using 'testMatch', as Jest's does in its defaults` -> `presets use 'testMatch', like Jest does in its defaults`